stable/rethinkdb/templates/_helpers.tpl

```diff
@@ -14,3 +14,21 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "rethinkdb.chart" -}}
```

Comment:
You copied this template but don't use it.

Reply:
You're incorrect, I've added this to the ServiceAccount entity, check the label `chart`

Reply:
You you just use this everywhere for consistency?

Reply:
yes

Reply:
done